File: sqlfs#/SqlFsLog.cs
using System;

namespace com.sss.sqlfs
{

	using Log = android.util.Log;

	/// <summary>
	///  A class for logging
	/// </summary>
	internal class SqlFsLog
	{
		private SqlFsLog()
		{
		}

		internal static void debug(Exception e)
		{
		   Log.d("SqlFsLog", e.Message);
		}

		internal static void debug(string eMsg)
		{
			Log.d("SqlFsLog", eMsg);
		}
	}

}

File: SqlFs/src/com/sss/sqlfs/SqlFsLog.java
package com.sss.sqlfs;

import android.util.Log;

/**
 *  A class for logging
 */
class SqlFsLog 
{
    private SqlFsLog() { }
    
	static void debug(Exception e) 
    {
       Log.d("SqlFsLog", e.getMessage());
    }

    static void debug(String eMsg) 
    {
    	Log.d("SqlFsLog", eMsg);
    }
}
